.github/workflows/build_micropython.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     if: ${{ github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name }}
     name: Build ${{ matrix.port }} port
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Sorry, but using "latest" from anything comes with the risk and in the future other 3rd party libraries will not be compatible with "latest" version... so please use ALWAYS specific version, which is 100% compatible with other libraries we use. 
I would say using ubuntu 22 is fine, or 24 if all the other libraries work with that, too (please test it).

Reply:
`master` already uses `ubuntu-latest` (see https://github.com/lvgl/lvgl/blob/master/.github/workflows/ccpp.yml#L12) but specifying the version really makes sense.

cc @W-Mai

Reply:
I am in full agreement with setting the Ubuntu version. I did this just to check because I had found the version for SDL for Ubuntu 24.04 and while it wasn't 2.30.1 which is where I knew 100% that the enumerations were in I wanted to see if it would fix the problem. Once I knew the problem was fixed I then went digging to see what version of SDL had the change made in it and what would be the minimum version of Ubuntu would have the changes in it. Keep this as a reminder that that any changes made that use SDL are going to have to work with SDL version 2.0.20 which is ~2 years old which is pretty outdated. 

The other option is to compile SDL then we can use a newer version if wanted. IDK why SDL is not updated like it should be for the Ubuntu releases. They stop releasing the packages for newer versions of SDL for the different Ubuntu releases.


As it sits right now we know that everything works with Ubuntu 24.04 without a doubt. 24.04 has not been released as an officially yet but it is smart to run tests for it because it is going to be coming up. So maybe modifying the builds to run tests for 22.04 and also 24.04.

I am not sure what to do about the SDL issue and then having such an old release to the Ubuntu package index. unfortunately they do not build a Debian package when they do their releases.

Reply:
@kisvegabor 

what was the solution for this change request?

Reply:
I've opened https://github.com/lvgl/lvgl/pull/6032 to see how Ubuntu 24.04 behaves.

Reply:
ok once that gets merged in I will update this PR and correct any merge conflicts.

Reply:
GitHub's CI don't want to queue some of the actions. I don't know if it's just a temporal issue, or related to ubuntu 24.04 somehow.